lib/base/json.cpp

```diff
@@ -207,11 +207,11 @@ String icinga::JsonEncode(const Value& value, bool pretty_print)
 
 Value icinga::JsonDecode(const String& data)
 {
-	String sanitized (Utility::ValidateUTF8(data));
+	Utility::ValidateUTF8(const_cast<String&>(data));
```

Comment:
![](https://c.tenor.com/2Qrxb60EYFoAAAAM/please-no-michael-scott.gif)

Reply:
Searched "why const cast is bad", first hit:

https://stackoverflow.com/a/10498371